### DISCUSSION TITLE
The possibility of printing the value upon "value is not a valid enumeration member" error message

### DISCUSSION THREAD

**Discussion by anilbey** [pydantic/pydantic#4960] [Question]
Hi, I think it would be really useful to print the value itself alongside the permitted values.
Does it make sense to print the actual value as well the permitted values?

It happens when an enum gets an unexpected value.

> 
> E   pydantic.error_wrappers.ValidationError: 1 validation error for RelativeShotNoise
> E   mode
> E     value is not a valid enumeration member; permitted: 'current', 'conductance' (type=type_error.enum; enum_values=[<ClampMode.CURRENT: 'current'>, <ClampMode.CONDUCTANCE: 'conductance'>])

**Comment by Ledorub** [comment]
#355 related.

**Comment by samuelcolvin** [comment]
This will be possible in V2 where the input value is available on the error.

**Reply by anilbey** [reply]
Thanks, I'm looking forward to V2
